src/components/general/Blog.vue

```diff
@@ -9,12 +25,122 @@ export default {
 	props: {
 		limit: Number
 	},
+	data() {
+		return {
+			blogData: []
+		}
+	},
+	methods: {
+		toText(htmlText) {
+			let plainText = document.createElement('div')
+			plainText.innerHTML = htmlText
+			htmlText = plainText.innerText
+			return htmlText
+		}
+	},
+	beforeMount() {
+		let referenceThis = this
+		let httpRequest = new XMLHttpRequest();
+		httpRequest.onreadystatechange = function() {
+			if (this.readyState == 4 && this.status == 200) {
+				referenceThis.blogData = JSON.parse(this.responseText);
+			}
+		};
+		httpRequest.open("GET", "http://test.cutc.ca/mediumPosts.php", true);
+		httpRequest.send();
+	}
 }
 </script>
 
 <style scoped>
+#blog {
+	max-width: 1120px;
+}
+
 a {
 	color: var(--main-color);
+}
+
+router-link {
 	text-decoration: underline;
 }
+
+.header {
+	display: flex;
+	align-items: baseline;
+	grid-column-gap: 40px;
+	font-weight: 600px;
+}
+
+.grid {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	gap: 5vh 10vw;
```

Comment:
Please make the grid gaps a constant size, and then change it once in a media query.  (Preferably smaller then they are now, we want to show off the articles!)